shared/locales/phone_types/phone_types.en-US.l20n

```diff
@@ -0,0 +1,25 @@
+/* Field labels */
+<phone_type {
+  *mobile:    "Mobile",
+  home:       "Home",
+  work:       "Work",
+  personal:   "Personal",
+  fax_home:   "Fax home",
+  fax_office: "Fax office",
+  fax_other:  "Fax other",
+  other:      "Other"
+}>
+<_phone_type[$type] {
+  *mobile:    "{{ phone_type.mobile }}",
+  home:      "{{ phone_type.home }}",
+  work:      "{{ phone_type.work }}",
+  personal:  "{{ phone_type.personal }}",
+  faxHome:   "{{ phone_type.fax_home }}",
+  faxOffice: "{{ phone_type.fax_office }}",
+  faxOther:  "{{ phone_type.fax_other }}",
+  other:     "{{ phone_type.other }}"
+}>
```

Comment:
OK, thanks to both for clarifying that; then I'll probably need 16 different strings here, 8 for the phone types (these can be alone) plus 8 for the phone-type-comma-carrier forms. I suppose I can reuse the strings for the phone types as described above.